service.elastic.yml

```diff
@@ -1,6 +1,6 @@
 ---
 
-docker_image: "pmce/ce-vm:{{ ce_vm_version }}"
+docker_image: "pmce/ce-vm-elastic-{{ elastic_version }}:{{ ce_vm_version }}"
```

Comment:
Right. For this to work, you'll need elastic_version in the config.yml file. The roles' default one won't be enough.